external/model-preparation-algorithm/mpa_tasks/apis/detection/task.py

```diff
@@ -344,6 +344,21 @@ def patch_color_conversion(pipeline):
                 elif pipeline_step.type == "MultiScaleFlipAug":
                     patch_color_conversion(pipeline_step.transforms)
 
+        def patch_data_pipeline(cfg):
+            pipeline = cfg.get("pipeline", None)
+            if pipeline is not None:
+                for pipeline_step in pipeline:
+                    if pipeline_step.type == "LoadImageFromFile":
+                        pipeline_step.type = "LoadImageFromOTEDataset"
+                    if pipeline_step.type == "LoadAnnotations":
+                        pipeline_step.type = "LoadAnnotationFromOTEDataset"
+                        pipeline_step.domain = domain
+                        pipeline_step.min_size = cfg.pop("min_size", -1)
+                    if subset == "train" and pipeline_step.type == "Collect":
+                        pipeline_step = BaseTask._get_meta_keys(pipeline_step)
```

Comment:
The `ignored_labels` key error in the comment is a problem that can be solved by going through this `_get_meta_keys` function. I can't seem to get into this line, so please check.